src/vibrato.ts

```diff
@@ -9,11 +9,8 @@ import { Tables } from './tables';
 import { Category } from './typeguard';
 
 export interface VibratoRenderOptions {
-  vibratoWidth: number;
-  waveWidth: number;
-  waveHeight: number;
-  waveGirth: number;
-  harsh: boolean;
+  code: number;
+  width: number;
 }
 
 /** `Vibrato` implements diverse vibratos. */
```

Comment:
Can we expand this comment?

I think we need to point developers to: https://w3c.github.io/smufl/latest/tables/multi-segment-lines.html